module2/service/src/main/java/com/epam/esm/service/validator/impl/OrderValidatorImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.esm.service.validator.impl;
+
+import com.epam.esm.service.dto.OrderCreateDto;
+import com.epam.esm.service.exception.ServiceException;
+import com.epam.esm.service.validator.OrderValidator;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import static com.epam.esm.service.exception.ErrorCode.ERROR_301400;
+import static com.epam.esm.service.exception.ErrorCode.ERROR_302400;
+
+@Slf4j
+@Component
+public class OrderValidatorImpl implements OrderValidator {
```

Comment:
It is a little bit strange that in controller you use Hibernate validation, but here you use custom validation